interfaces/builtin/upower_observe.go

```diff
@@ -123,13 +122,12 @@ socket AF_NETLINK - NETLINK_KOBJECT_UEVENT
 `
 
 const upowerObservePermanentSlotDBus = `
-<!-- DBus policy for upower (based on upstream version 0.99.4) -->
+<!-- From upstream version 1.90.0 -->
+<!-- Only root can own the service -->
 <policy user="root">
   <allow own="org.freedesktop.UPower"/>
 </policy>
 <policy context="default">
-  <deny own="org.freedesktop.UPower"/>
```

Comment:
I can't see a good reason why this `deny` line has been removed - and it makes me concerned that then the default policy doesn't stop others from claiming this name - can this be re-added? Or can you explain why it has been removed?

Reply:
It can probably be re-added. I only copied the content of what was provided by 1.90.0.

I will test that.

Reply:
That works. Reverted.